### ISSUE TITLE
This add-on has been upgraded. ... there is no upgrade procedure defined for this addon...

### ISSUE THREAD

**Issue by djowett** [miohtama/visualtitle#6]
This error appears on the addons section of the plone control panel, when upgrading the product (to 0.3)

```
This add-on has been upgraded. Old profile version was unknown. 
New profile version is 0.3. 
There is no upgrade procedure defined for this add-on. 
Please consult the add-on documentation for upgrade information, 
or contact the add-on author. 
```

This isn't something to worry about, because there is no need for any upgrade step.

I think the warning occurs because there is not (was never) a metadata.xml for the product, but now I guess we need an "upgrade step" to clear this away.

**Comment by miohtama** [comment]
Does this prevent installation?

**Comment by djowett** [comment]
No, and "uninstall, install" through Plone Control Panel clears it.
But re-install the portal_quickinstaller way doesn't.  Do we just leave this to appease worried users?
